### PR TITLE
Travis CI configuration

### PR DESCRIPTION
**Pull request by honzajavorek**
I know almost nothing about Go, so the setup will probably need some more work. @pksunkara, could you please help me with this?

Status of this branch can be seen here:
- https://github.com/apiaryio/goodman/branches
- https://travis-ci.org/apiaryio/goodman

There are some docs on this: https://docs.travis-ci.com/user/languages/go

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: go
+
+before_install:
+  - npm install -g dredd
+  - cd cucumber
+  - bundle install
+
+script:
+  - cd cucumber
+  - bundle exec cucumber
```
Comment (pksunkara) [inline .travis.yml:10]: There are no Go tests at all which is why the CI is failing. If you want to test only this, you can probably use ruby.

Comment (honzajavorek) [inline .travis.yml:10]: But shouldn't the cucumber test the Go app? I'll try Ruby, thanks!

**Comment by honzajavorek** [comment]
The tests seem to run correctly now once I switched the language from Go to Ruby, but they are failing:

```
10 scenarios (10 failed)
83 steps (10 failed, 44 skipped, 29 passed)
```

**Comment by netmilk** [comment]
Great stuff guys! Thank you very much! 

@snikch Would you mind merging? We're working on Dredd dependant builds for all handlers to automatically verify they're working. It will be part of Dredd's QA on master before each release.

File: .travis.yml (hunk 1 above)
Comment (snikch) [inline .travis.yml:10]: The cucumber test should run the `go build` command, which in turn is what is tested.

**Comment by snikch** [comment]
This makes sense to get the tests running, so I'll merge. Will need to look at tests passing as a separate issue (as part of my ongoing API work I'll get to this, but I have no ETA).

**Comment by snikch** [comment]
@netmilk Merged :ok_hand:

**Comment by honzajavorek** [comment]
Thanks!

**Pull request merged**